certified-connectors/Power Textor/apiDefinition.swagger.json

```diff
@@ -0,0 +1,6782 @@
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "PlanId": {
+                        "type": "string",
+                        "description": "PlanId"
+                      },
+                      "Order": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "Order"
+                      },
+                      "Name": {
+                        "type": "string",
+                        "description": "Name"
+                      },
+                      "Package": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "Package"
+                      },
+                      "MonthlyPrice": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "MonthlyPrice"
+                      },
+                      "OveragePrice": {
+                        "type": "number",
+                        "format": "float",
+                        "description": "OveragePrice"
+                      },
+                      "StripeMonthlyPlanId": {
+                        "type": "string",
+                        "description": "StripeMonthlyPlanId"
+                      },
+                      "StripeOveragePlanId": {
+                        "type": "string",
+                        "description": "StripeOveragePlanId"
+                      },
+                      "Public": {
+                        "type": "boolean",
+                        "description": "Public"
+                      },
+                      "PeriodStart": {
+                        "type": "string",
+                        "description": "PeriodStart"
+                      },
+                      "PeriodEnd": {
+                        "type": "string",
+                        "description": "PeriodEnd"
+                      },
+                      "Usage": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "Usage"
+                      }
+                    }
+                  },
+                  "description": "Data"
+                },
+                "Success": {
+                  "type": "boolean",
+                  "description": "Success"
+                },
+                "Error": {
+                  "type": "string",
+                  "description": "Error"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Plans",
+        "description": "Get Plans",
+        "operationId": "GetPlans",
+        "parameters": [],
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/api/user": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
```

Comment:
@MohsinAQ Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**

Reply:
Can you please explain in more detail or give an example of how it is correctly implemented?
We have added "200" response code, what else do we have to add?

Reply:
One of the examples how the response should look like:
 "responses": {
          "200": {
            "description": "success",
            "schema": {
              "type": "string",
              "title": "SuccessStatus",
              "format": "string"
	    }
	  }